tests/cis_tests/leftovers_script.py

```diff
@@ -1,31 +1,67 @@
-from snapred.backend.recipe.algorithm.data.WrapLeftovers import WrapLeftovers
-from snapred.backend.recipe.algorithm.data.ReheatLeftovers import ReheatLeftovers
-
+import snapred.backend.recipe.algorithm
 from mantid.simpleapi import *
 from mantid.testing import assert_almost_equal as assert_wksp_almost_equal
 
+from time import time
+
 
 # Load focussed data
-Load(Filename='/SNS/users/wqp/SNAP/shared/Calibration_dynamic/Powder/04bd2c53f6bf6754/normalization/v_0014/tof_column_s+f-vanadium_058810_v0014.nxs', OutputWorkspace='raw')
+# Load(Filename='/SNS/users/wqp/SNAP/shared/Calibration_dynamic/Powder/04bd2c53f6bf6754/normalization/v_0014/tof_column_s+f-vanadium_058810_v0014.nxs', OutputWorkspace='raw')
```

Comment:
not sure if you want to keep this here to be uncommented later

Reply:
That was intended to be there, in case it needs to be uncommented later.  But it prevented  non-Michael people from running the script